servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -183,8 +205,12 @@ private State(final ServiceDescriptorProto serviceProto) {
     }
 
     void generate(final ServiceDescriptorProto serviceProto) {
-        final State state = new State(serviceProto);
-        final TypeSpec.Builder serviceClassBuilder = context.newServiceClassBuilder(serviceProto);
+        final String name = context.deconflictJavaTypeName(
+                sanitizeIdentifier(serviceProto.getName(), false) + Service);
+        final State state = new State(serviceProto, name);
+
+        final TypeSpec.Builder serviceClassBuilder = context.newServiceClassBuilder(serviceProto)
+                .addJavadoc("Class for $L", serviceProto.getName());
```

Comment:
This javadoc does not provide much value. Consider rewriting (TBH, no idea how to make it more meaningful, maybe reference the `*.proto` file this class is generated for) or removing it.